lidar:
  T_lidar_cam1:
  - [ 0.1024,  0.0761, 0.9918,  0.2395]
  - [-0.9945,  0.0289, 0.1004, -0.0417]
  - [-0.0210, -0.9967, 0.0786, -0.1229]
  - [ 0.0000,  0.0000, 0.0000,  1.0000]
cam0:
  T_cam_imu:
  - [0.013687134250334743, 0.9998915394196658, -0.005437991631561358, -0.0007042041653323161]
  - [-0.9998220688141216, 0.013615177482173446, -0.0130559432588893, -0.0015861692685288122]
  - [-0.012980487982496344, 0.005615722491410728, 0.9998999802943475, -0.017815961182110547]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [1]
  camera_model: pinhole
  distortion_coeffs: [-0.048031442223833355, 0.011330957517194437, -0.055378166304281135,  0.021500973881459395]
  distortion_model: equidistant
  intrinsics: [226.38018519795807, 226.15002947047415, 173.6470807871759, 133.73271487507847]
  resolution: [346, 260]
  rostopic: /davis/left/image_raw
cam1:
  T_cam_imu:
  - [0.002658051013083418, 0.9999459669775138, -0.010049770654903653, -0.10052351879548456]
  - [-0.9999921475216119, 0.002628365498611729, -0.0029659045702786734, -0.002149609013368439]
  - [-0.0029393298430320275, 0.01005757526494452, 0.9999451012529955, -0.01780039434280798]
  - [0.0, 0.0, 0.0, 1.0]
  T_cn_cnm1:
  - [0.9999285439274112, 0.011088072985503046, -0.004467849222081981, -0.09988137641750752]
  - [-0.011042817783611191, 0.9998887260774646, 0.01002953830336461, -0.0003927067773089277]
  - [0.004578560319692358, -0.009979483987103495, 0.9999397215256256, 1.8880107752680777e-06]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [-0.04846669832871334, 0.010092844338123635, -0.04293073765014637,    0.005194706897326005]
  distortion_model: equidistant
  intrinsics: [226.0181418548734, 225.7869434267677, 174.5433576736815, 124.21627572590607]
  resolution: [346, 260]
  rostopic: /davis/right/image_raw
